### PR TITLE
Events API for calendar

### PR DESCRIPTION
**Pull request by rhnonose**
- Change the json routers to stay inside `/api`
- Add `fetch` action to get:
  - All events
  - Events added

### CODE REVIEW

**Comment by ghatighorias** [comment]
is the purpose creating standalone separate api ( calendar and events separated ) ?

**Comment by rhnonose** [comment]
@ghatighorias there's definitely some common functionalities in the mix, we have to think about it. Do you have any preference? Keeping it separate would make it easy to also have filters for "show only classes"/"show only events" (at least for me because we can do that in the backend rather than in js 😅 )

**Comment by ghatighorias** [comment]
I am okay to keep them separate just that maybe we can move the common functionality to another module. we can however on the other hand add a card for this as a refactoring for later too

**Comment by rhnonose** [comment]
Agreed, card created.

File: lib/course_planner_web/views/event_view.ex [hunk 1]
```diff
@@ -28,4 +28,29 @@ defmodule CoursePlannerWeb.EventView do
...
+  def render("event.json", %{event: event}) do
+    %{
+      name: event.name,
+      description: event.description,
+      location: event.location,
+
+      date: event.date,
+      starting_time: event.starting_time,
+      finishing_time: event.finishing_time,
+      users: Enum.map(event.users, &user_json/1)
```
[the remark this answers is not in the document]
Comment (rhnonose) [reply lib/course_planner_web/views/event_view.ex]: I'm not sure I understand the question.

**Pull request merged**